census-api/src/main/scala/hmda/census/validation/CensusValidation.scala

```diff
@@ -9,6 +9,10 @@ object CensusValidation {
     indexedTract.contains(tract)
   }
 
+  def isTractValidTest(tract: String): Boolean = {
```

Comment:
@omnipresent07 does this need to be implemented? Looks like it's just a placeholder to default to true